fedimint-dummy-client/src/db.rs

```diff
@@ -19,6 +29,132 @@ pub struct DummyClientFundsKeyV0;
 
 impl_db_record!(
     key = DummyClientFundsKeyV0,
+    value = (),
+    db_prefix = DbKeyPrefix::ClientFunds,
+);
+
+#[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash)]
+pub struct DummyClientFundsKeyV1;
+
+impl_db_record!(
+    key = DummyClientFundsKeyV1,
     value = Amount,
     db_prefix = DbKeyPrefix::ClientFunds,
 );
+
+#[derive(Debug, Clone, Encodable, Decodable, Eq, PartialEq, Hash)]
+pub struct DummyClientNameKey;
+
+impl_db_record!(
+    key = DummyClientNameKey,
+    value = String,
+    db_prefix = DbKeyPrefix::ClientName,
+);
+
+/// Migrates the database from version 0 to version 1 by
+/// removing `DummyClientFundsKeyV0` and inserting `DummyClientFundsKeyV1`.
+/// The new key/value pair has an `Amount` as the value.
+pub async fn migrate_to_v1(
```

Comment:
Thought (no need to do it in this PR): we would probably be better of if the example module was `fedimint-module-empty` and not `dummy` module.